library/modules/Textures.cpp

```diff
@@ -142,6 +165,34 @@ long Textures::getTexposByHandle(TexposHandle handle) {
     return -1;
 }
 
+TexposHandle Textures::createTile(std::vector<uint32_t>& pixels, int tile_px_w, int tile_px_h) {
+    auto texture = create_texture(pixels, tile_px_w, tile_px_h);
+    auto handle = Textures::loadTexture(texture);
+    return handle;
+}
+
+std::vector<TexposHandle> Textures::createTileset(std::vector<uint32_t>& pixels, int texture_px_w,
+                                                  int texture_px_h, int tile_px_w, int tile_px_h) {
+    auto texture = create_texture(pixels, texture_px_w, texture_px_h);
+    auto handles = slice_tileset(texture, tile_px_w, tile_px_h);
+    return handles;
+}
+
+void Textures::deleteHandle(TexposHandle handle) {
+    auto texpos = Textures::getTexposByHandle(handle);
+    if (texpos > 0)
+        delete_texture(texpos);
+    if (g_handle_to_texpos.contains(handle))
+        g_handle_to_texpos.erase(handle);
+    if (g_handle_to_surface.contains(handle)) {
+        auto surface = g_handle_to_surface[handle];
+        for (auto i = 0; i < surface->refcount; i++) {
+            DFSDL_FreeSurface(surface);
+        }
```

Comment:
does `surface->refcount` change when `DFSDL_FreeSurface` is called? If so, then this should be
```cpp
while (surface->refcount)
    DFSDL_FreeSurface(surface);
```